docs/SCAFFOLDING.md

```diff
@@ -0,0 +1,76 @@
+# C++ Scaffolding
+
+## Prerequisites
+
+To be able to generate the C++ scaffolding, only a `.udl` file is needed, to learn more about how the UDL syntax works, refer to the [official documentation](https://mozilla.github.io/uniffi-rs/Overview.html) from Mozilla.
+
+## Scaffolding generation
+
+To generate the C++ scaffolding you can run:
+
+```bash
+uniffi-bindgen-cpp --scaffolding your_definitions.udl --out-dir output_directory
+```
+
+And it will generate the scaffolding the provided location.
```

Comment:
```suggestion
And it will generate scaffolding in the provided location.
```

Reply:
Maybe you could also list the files that will be generated. Its just 1 C++ file, right?

Reply:
The file is mentioned in the `Scaffolding integration` section, but maybe it would make sense to move it into this section instead